src/netaddress.cpp

```diff
@@ -8,14 +8,18 @@
 #include "config/bitcoin-config.h"
 #endif
 
-#include "netaddress.h"
-#include "hash.h"
-#include "utilstrencodings.h"
-#include "tinyformat.h"
+#include <netaddress.h>
+#include <hash.h>
+#include <utilstrencodings.h>
+#include <util/asmap.h>
+#include <tinyformat.h>
 
 static const unsigned char pchIPv4[12] = { 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0xff, 0xff };
 static const unsigned char pchOnionCat[] = {0xFD,0x87,0xD8,0x7E,0xEB,0x43};
 
+// 0xFD + sha256("bitcoin")[0:5]
```

Comment:
Adds maintenance pressure to change it, and there are zero benefits to do so. Therefore, I'd recommend against it. 

I.e. change it here, then also needs to be changed for 1.21 here: https://github.com/dogecoin/dogecoin/blob/56b3c4994aa538a96f1828e967fc6ef60934e6ac/src/netaddress.h#L85-L92, and every time someone has the great idea to re-port into a newer bitcoin core, this will be yet another task on the list that can be missed, screwing with upgrade processes, backward compatibility and so on.

Reply:
This is only used dogecoind-internal, uniquely identifying a guaranteed-to-be-invalid hostname (not IP-address) to be able to keep track of the ***source*** of addresses obtained through DNS and fixed seeds in addrman. The "namespace" bytes after `0xfd` to identify invalidate these are completely arbitrary, but should be consistent between versions that read `peers.dat`, now and in the future. So the best solution is to not change them, imho.

See https://github.com/bitcoin/bitcoin/pull/10446 for why this was done (note that we may implement this whole change set too at some point, but I think it's a minor thing so it doesn't have to block anything.)